packages/@ama-sdk/core/src/plugins/mgw-mdw-auth/mgw-mdw-auth.request.ts

```diff
@@ -27,17 +24,14 @@ export async function sha256(value: string) {
  * @param value Value to hash
  * @param secretKey Secret cryptographic key
  */
-export function hmacSHA256(value: string, secretKey: string) {
```

Comment:
This function was exported. Do we risk breaking changes because of the `async`?

Reply:
This function is exported to be tested, it does not make sense to be used by itself

Reply:
maybe it should be exported in a different file then, to avoid having it in the barrel

Reply:
In term of breaking change it is even worse and will appear as function removal.
But ok, let's do it, soons like the best compromise :)